pages/linux/mons.md

```diff
@@ -0,0 +1,19 @@
+# mons
+
+> quickly manage 2-monitors display
```

Comment:
```suggestion
> A tool to manage 2-monitors displays.
```

Reply:
Actually "2-monitors displays" is a little redundant. I think we should find another way to phrase this.